openhtf/util/measurements.py

```diff
@@ -93,16 +95,23 @@ class MultipleValidatorsException(Exception):
   """Multiple validators were used when defining a measurement."""
 
 
+class Outcome(Enum):
+  """Enum values for measurement 'outcome' fields."""
+  PASS = 'PASS'
```

Comment:
Ah yeah, pylint complains about bad-name that way, but it's more concise and cleaner, so i'll go with that, good point.